content/7-webapp-features/1-fetch-data/ember/users.js

```diff
@@ -0,0 +1,19 @@
+import Route from '@ember/routing/route';
+import fetch from 'fetch';
+
+export default class UsersRoute extends Route {
+	model = async () => {
+		try {
+			const response = await fetch('https://randomuser.me/api/?results=3');
+			if (!response.ok) {
+				if (response.status == 401) {
+					throw new Error('Please try again');
+				}
```

Comment:
Why handling HTTP 401 ? Other examples in React or Svelte don't handle this error.